src/System.Diagnostics.EventLog/tests/EventLogTests/EventLogTests.cs

```diff
@@ -349,5 +349,15 @@ public void GetMessageUsingEventMessageDLL()
                 Assert.Contains("", eventlog.Entries.LastOrDefault()?.Message ?? "");
             }
         }
+
+        [ConditionalFact(typeof(Helpers), nameof(Helpers.SupportsEventLogs))]
+        public void GetEventLogEntriesTest()
+        {
+            foreach (var eventLog in EventLog.GetEventLogs())
+            {
+                // Accessing eventlog properties should not throw.
+                int _ = eventLog.Entries.Count;
```

Comment:
>int _ = eventLog.Entries.Count; [](start = 16, length = 31)

is it possible the compiler can optimize this line and not execute it at all?

would it better to just Assert.Trye(eventLog.Entries.Count >= 0)